### DISCUSSION TITLE
Labels, Tag and Badges

### DISCUSSION THREAD

**Discussion by Porlin** [twilio-labs/paste#1564] [Q&A] [answered]
Hey Team,

I was wondering where you currently are at in regards to labels, tags and badges? I have some pages to update in the Flex Console that uses the tag/label - please see attached.

Should we continue to use the existing component in this case? 

Many thanks,
Porlin

<img width="1211" alt="Screenshot 2021-06-14 at 18 26 04" src="https://user-images.githubusercontent.com/49287701/121933838-7cc6c580-cd3e-11eb-9afb-ba52353979af.png">

**Comment by richbachman** [comment]
Hi @Porlin. Would you mind clarifying which part of the screenshot you're asking about. I'm assuming it's this: https://share.getcloudapp.com/12u4rPvD, but wanted to clarify. Thanks!

**Reply by Porlin** [reply]
Yes, that's the one @richbachman

**Comment by TheSisb** [comment, marked as answer]
Hi @Porlin,

Looks like the `Pill` component from the screenshot above will be coming in the 2021-08-24 release (https://paste.twilio.design/roadmap).

If possible, you can compose a component using Paste tokens and the Box component. If you do that, please share your work here so we can add it to our research/kickoff work when we start on that component to make your transition easier.  If that composition isn't possible, you can keep using what you already have until the component is ready as well.
